projects/2024/neutron-ball/report-analytical.ipynb

```diff
@@ -146,16 +146,17 @@
     "Solve the heterogeneous spatial neutron flux magnitude diffusion problem. Find the neutron spatial flux magnitude, $u_1:[0,R]\\subset\\Reals\\rightarrow\\Reals$, $u_2:[R,R_\\infty]\\subset\\Reals\\rightarrow\\Reals$,\n",
     "such that:\n",
     "\n",
+    "\n",
     "\\begin{align*}\n",
-    " -\\frac{1}{r^2}\\bigl(-D_1\\, r^2 u_1'\\bigr)'(r) - \\Sigma_{a,1}\\,u_1(r) + s_{0,1} &= 0 \\quad\\quad \\forall \\quad\\quad r\\in\\ ]0,R[, \\\\\n",
```

Comment:
Eliminating the small space makes the equation look odd since the diffusion coefficient shows up too close to the unknown variable. Proper typesetting calls for a small space to make the distinction. Note the difference $D_1\\,r^2 u_1$ and $D_1r^2u_1$. The former is better.